cmd/sandbox-ce/cmd/web.go

```diff
@@ -0,0 +1,79 @@
+package cmd
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/pkg/browser"
+	"github.com/pkg/errors"
+	"github.com/src-d/superset-compose/cmd/sandbox-ce/compose"
+)
+
+type webCmd struct {
+	Command `name:"web" short-description:"Web"`
```

Comment:
Let's add both short and long descriptions. Something like "Open the web interface on your browser"